### DISCUSSION TITLE
Where are the Registry keys for the "Run sandboxed" context menu items?

### DISCUSSION THREAD

**Discussion by e-t-l** [sandboxie-plus/Sandboxie#3544] [Q&A Feedback]
According to [./Installer/Sandboxie-Plus.iss#L93](https://github.com/sandboxie-plus/Sandboxie/tree/master/Installer/Sandboxie-Plus.iss#L93) the registry keys for the "Run sandboxed" shell entries should be at `HKCU\Software\Classes\*\shell\sandbox` yet after enabling the feature in Sandboxie-Plus v1.12.6, these registry keys do not exist. Am I crazy? Where the heck do I find these keys?

(context for the question: I'd like to change the placement of it in the menu, just for my personal preference.)

**Comment by offhub** [comment]
This may be because the context menu has changed in Windows 11 and `\shell\` locations are no longer used for Windows 11. [****](https://github.com/sandboxie-plus/Sandboxie/commit/f76929f0359efc2a336f2e6823a750f4ce7f0bf9)

> ## [1.2.7 / 5.57.7] - 2022-07-31
> 
> - setup of shell integration is now done by SandMan, not the installer
